frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java

```diff
@@ -303,8 +304,8 @@ private void setBranchToolTipText(IManagedBranchSnapshot branch) {
   }
 
   private static String getSyncToParentStatusBasedToolTipText(INonRootManagedBranchSnapshot branch) {
-    val currentBranchName = branch.getName();
-    val parentBranchName = branch.getParent().getName();
+    val currentBranchName = escapeHtml4(branch.getName());
+    val parentBranchName = escapeHtml4(branch.getParent().getName());
     return Match(branch.getSyncToParentStatus()).of(
         Case($(InSync),
             getString("string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-parent-status-tooltip.in-sync.HTML")
```

Comment:
I'd also consider marking the varargs param of `GitMacheteBundle#format` as `@Untainted`

Reply:
+1, got the same idea independently right now, only to see this comment :trollface: